packages/p2p/src/pages/advertiser-page/advertiser-page.jsx

```diff
@@ -319,82 +320,84 @@ const AdvertiserPage = () => {
-                                            />
-                                        </div>
-                                        <div className='advertiser-page__rating--row'>
-                                            <Text
-                                                className='advertiser-page__joined-since'
-                                                color='less-prominent'
-                                                size='xs'
-                                            >
-                                                {joined_since ? (
-                                                    <Localize
-                                                        i18n_default_text='Joined {{days_since_joined}}d'
-                                                        values={{ days_since_joined: joined_since }}
-                                                    />
-                                                ) : (
-                                                    <Localize i18n_default_text='Joined today' />
-                                                )}
-                                            </Text>
-                                        </div>
-                                    </React.Fragment>
-                                </DesktopWrapper>
-                                {rating_average ? (
-                                    <React.Fragment>
-                                        <div className='advertiser-page__rating--row'>
-                                            <Text color='prominent' size={isMobile() ? 'xxxs' : 'xs'}>
-                                                {rating_average_decimal}
-                                            </Text>
-                                            <StarRating
-                                                empty_star_className='advertiser-page__rating--star'
-                                                empty_star_icon='IcEmptyStar'
-                                                full_star_className='advertiser-page__rating--star'
-                                                full_star_icon='IcFullStar'
-                                                initial_value={rating_average_decimal}
-                                                is_readonly
-                                                number_of_stars={5}
-                                                should_allow_hover_effect={false}
-                                                star_size={isMobile() ? 17 : 20}
-                                            />
-                                            <div className='advertiser-page__rating--text'>
-                                                <Text color='less-prominent' size={isMobile() ? 'xxxs' : 'xs'}>
-                                                    {rating_count === 1 ? (
+                            {!isEmptyObject(info) && (
```

Comment:
inorder to avoid the glitch for empty case. could provide a skeleton loader if needed in future.